action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
   community_year:
     description: 'Year of the community release (2020, 2019)'
     default: '2020'
+  cache:
```

Comment:
This should probably be named `install_dir`, and then the doc explains how to cache this install dir with `actions/cache`.

With the name `cache` it sounds like there is nothing else to do for the install to be cached,